### DISCUSSION TITLE
Setting up the development environment

### DISCUSSION THREAD

**Discussion by cxcubehd** [nxrighthere/UnrealCLR#214] [Ideas]
### Introduction
I am coming from Unity wanting to switch to Unreal Engine, as I don't really like how slow the Unity Editor is and ect. I have worked with C++ before, but I don't like it for those reasons:
- Hard to maintain
- #include errors (circular dependencies as an example)
- Linker errors
- and ect.

I don't really want to argue here wether C++ is good or not. So that's why I want to use C# for my project, as it is more readable and easy to maintain. The problem I have right now is that the process of needing to compile my project manually every time, in order to see my results. I know that you could integrate it somehow with the IDE, but this process is just to much work for a simple test project. 

### Here's what I would like to see
- More details about the project setup in the manual
for example that you must restart the project in order to be able to use Unreal CLR
- How to setup comon IDEs for this (with run/build configurations), especially for VS and Rider

**Comment by nxrighthere** [comment]
At some point, I'll cover this information for Visual Studio Code at least. I think it would be a good example for setting up similar IDEs.
